library/types/src/modules/URLRecode.rb

```diff
@@ -9,7 +9,7 @@ module Yast
   class URLRecodeClass < Module
     # these will be substituted to a regex character class
     USERNAME_PASSWORD_FRAGMENT_SAFE_CHARS = "-A-Za-z0-9_.!~*'()".freeze
-    PATH_SAFE_CHARS =                       "-A-Za-z0-9_.!~*'()/:".freeze
+    PATH_SAFE_CHARS =                       "-A-Za-z0-9_.!~*'()/:$".freeze
     QUERY_SAFE_CHARS =                      "-A-Za-z0-9_.!~*'()/:=&".freeze
```

Comment:
Just a dumb question: Would it make sense to add `$` to those other `_SAFE_CHARS` as well? Or is it really exclusively usefull for `PATH_SAFE_CHARS`?

Reply:
According to the RFC, it is valid for `query` part to, but not for `userinfo` (part of `authority`). I was simply keeping changes at minimum.

Reply:
Usually the variables are only used in the path part, in theory it could be included in the query but I have never seen that. For that you would need to implement some server side processing logic instead of just simply serving static files...